### PR TITLE
[ISSUE-387][DOC] Doc about worker status recover after restart

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
截屏2022-08-24 下午8 21 53

截屏2022-08-24 下午8 21 59

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture @FMX

**Comment by AngersZhuuuu** [comment]
ping @waitinfuture @FMX

File: CONFIGURATION_GUIDE.md [hunk 1]
```diff
@@ -63,20 +63,28 @@ memory. Empirically, RSS worker off-heap memory should be set to `(numDirs  * bu
...
+|           rss.worker.recoverPath            | `${System.getProperty("java.io.tmpdir")}/recover` |                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| rss.worker.partitionSorterCloseAwaitTimeMs  |                     false                         |                                                                                                                                                            The wait time of waiting for sorting partition files during worker graceful shutdown.                                                                                                                                                                  |
```
Comment (waitinfuture) [inline CONFIGURATION_GUIDE.md:83]: default to 120s, and the config name should be rss.worker.partitionSorterCloseAwaitTime

Comment (AngersZhuuuu) [reply CONFIGURATION_GUIDE.md]: > rss.worker.partitionSorterCloseAwaitTime

Done

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**